OrangeBeans/Models/Payment/QuantityPromoDecorator.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OrangeBeans.Models
+{
+    public class QuantityPromoDecorator : PriceDecorator
+    {
+        private IPriceProcessor priceProcessor;
+        public List<ProductQuantityPromo> QuantityPromos { get; set; }
+
+        public QuantityPromoDecorator(IPriceProcessor priceProcessor)
+        {
+            this.priceProcessor = priceProcessor;
+        }
+
+        public override float CalculateOrderTotal(Dictionary<Product, int> products)
```

Comment:
As a consumer of this method, how can I know what is the `int` in the dictionary